aspnet/getting-started/installing-on-linux.rst

```diff
@@ -81,13 +81,95 @@ To build libuv you should do the following::
 .. note::
 
     ``make install`` puts ``libuv.so.1`` in ``/usr/local/lib``, in the above commands ```ldconfig`` is used to update ``ld.so.cache`` so that ``dlopen`` (see ``man dlopen``) can load it. If you are getting libuv some other way or not running ``make install`` then you need to ensure that dlopen is capable of loading ``libuv.so.1``.
-    
+
 Installing on CentOS, Fedora and derivatives
+--------------------------------------------
+
+The following instructions were tested using CentOS 7.
+
+Install the .NET Version Manager (DNVM)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Use the .NET Version Manager (DNVM) to install different versions of the .NET Execution Environment (DNX) on Linux.
+
+1. Install ``unzip`` if you don't already have it::
+
+    sudo yum install unzip
+
+2. Download and install DNVM::
+
+    curl -sSL https://raw.githubusercontent.com/aspnet/Home/dev/dnvminstall.sh | DNX_BRANCH=dev sh && source ~/.dnx/dnvm/dnvm.sh
+    sudo chmod +x ~/.dnx/dnvm/dnvm.sh
+    ~/.dnx/dnvm/dnvm.sh
```

Comment:
This file does not need to be executable. It cannot be run like a normal bash script, it doesn't work that way.

You have to source the script so that it is part of your shell for DNVM to work the way it is supposed to.